blockchain/blockdao.go

```diff
@@ -700,3 +822,11 @@ func deleteActions(dao *blockDAO, blk *block.Block, batch db.KVStoreBatch) error
 
 	return nil
 }
+
+// getDBIndex get db index from block height
+func getDBIndex(hei uint64, split uint64) int {
+	if split == 0 {
```

Comment:
Is it to support when SplitDBLength is not set (default 0), all data will be written into the same db?

Reply:
sure,now default is 100000,I'll change to 0